js/header.js

```diff
@@ -1,4 +1,5 @@
-const headerShrinkWidth = 1290;
+const headerShrinkWidth = 1319;
+var miniheader = document.getElementById('title-mini');
```

Comment:
Use const for consistency, also I would call this `miniTitle`

Reply:
done